services/QuillLMS/client/app/bundles/Diagnostic/utils/saveSession.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react'
```

Comment:
I love consolidating this logic into a shared util.  Nice to clean up the duplication.